api-report/odsp-driver-definitions.api.md

```diff
@@ -159,6 +159,8 @@ export enum OdspErrorType {
     // (undocumented)
     fetchTokenError = "fetchTokenError",
     // (undocumented)
+    fileStorageDomainChange = "fileStorageDomainChange",
```

Comment:
Maybe make it more generic, like use "redirect" wording?
Another thing to consider (@markfields - FYI) - we should either start prefixing these strings with "odsp", or start moving all of them into one global drive list.